openquake/hazardlib/gsim/base.py

```diff
@@ -273,7 +273,7 @@ def make_contexts(self, site_collection, rupture, filter=True):
 
     def filter_ruptures(self, src, sites):
         """
-        :param src: a source object
+        :param src: a source object, already filtered and split
         :param sites: a FilteredSiteCollection
         :param mon: a Monitor instance
```

Comment:
Please remove the param `mon` from the docstring

Reply:
Done